pandera/checks.py

```diff
@@ -286,3 +288,298 @@ def __repr__(self):
         name = getattr(self.fn, '__name__', self.fn.__class__.__name__)
         return "<Check %s: %s>" % (name, self.error) \
             if self.error is not None else "<Check %s>" % name
+
+    @staticmethod
+    def greater_than(min_value) -> 'Check':
```

Comment:
I see using a string is the standard practice: https://www.python.org/dev/peps/pep-0484/#forward-references

good to know!

Reply:
Yes, I learned it the hard way when I just wrote it without quotes.